install_config/monitoring/configuring-openshift-cluster-monitoring.adoc

```diff
@@ -53,6 +53,12 @@ The Monitoring stack is installed with {product-title} by default. You can preve
 
 `openshift_cluster_monitoring_operator_install`
 
+You can do it by running:
+
+----
+$ ansible-playbok -i <hosts> -e openshift_cluster_monitoring_operator_install=false /usr/share/ansible/openshift-ansible/playbooks/openshift-monitoring/config.yml
```

Comment:
this is fine, but I think it is better change to
$ ansible-playbook [-i </path/to/inventory>] <OPENSHIFT_ANSIBLE_DIR>/playbooks/openshift-monitoring/config.yml  -e openshift_cluster_monitoring_operator_install=False

Then it is consistent with 
https://docs.openshift.com/container-platform/3.11/install_config/cluster_metrics.html#metrics-cleanup

Reply:
Thanks @juzhao , implemented.

Reply:
I've also added another note mentioning `/usr/share/ansible/openshift-ansible` as the common case for `OPENSHIFT_ANSIBLE_DIR`.